core/src/main/resources/lib/form/select/select.js

```diff
@@ -43,6 +43,10 @@ function updateListBox(listBox,url,config) {
 Behaviour.specify("SELECT.select", 'select', 1000, function(e) {
 
         function hasChanged(selectEl, originalValue) {
+            if (selectEl.options.length == 0) {
+                // There are no options to select, so there is no possible change
```

Comment:
What about select boxes that have their contents filled via AJAX, but define a default value? I think credentials selection works like this.